### PR TITLE
test(input): add unit test for ui-input

### PR DESCRIPTION
**Pull request by larsdouweschuitema**
Add initial unit test for ui-input and check prop on ui-select

### CODE REVIEW

File: src/__tests__/ui-input.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            slots: {
+                prefix: SLOT_PREFIX,
+                sufix: SLOT_SUFIX,
+            },
+        });
+        input = wrapper.find('input');
+    });
+
+    it('Set value', () => {
+        input.setValue('Amsterdam');
```
Comment (renancarvalho) [inline src/__tests__/ui-input.test.js:20]: If theres no business logic on setting a value on the input.. this test is not necessary.

File: src/__tests__/ui-input.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        expect(input.element.value).toBe('Amsterdam');
+    });
+
+    it('Is valid', async () => {
```
Comment (renancarvalho) [inline src/__tests__/ui-input.test.js:24]: I would rewrite to something like.. "Should add invalid class if the component state is invalid..."

File: src/__tests__/ui-input.test.js (hunk 1 above)
Comment (larsdouweschuitema) [reply src/__tests__/ui-input.test.js:20]: - [ ] Remove

File: src/__tests__/ui-input.test.js (hunk 2 above)
Comment (larsdouweschuitema) [reply src/__tests__/ui-input.test.js:24]: - [ ] Use more descriptive text in test descriptions

**Review by renancarvalho — DISMISSED** [review]

**Review by renancarvalho — APPROVED** [review]

**Pull request merged**